### ISSUE TITLE
op type not registered 'DenseToDenseSetOperation' when running tfcompile

### ISSUE THREAD

**Issue by mehditlili** [tensorflow/tensorflow#21675]
### System information
- **Have I written custom code (as opposed to using a stock example script provided in TensorFlow)**: Yes (code from maskrcnn, and export.py to freeze my model)
- **OS Platform and Distribution (e.g., Linux Ubuntu 16.04)**: Ubuntu 16.04
- **Mobile device (e.g. iPhone 8, Pixel 2, Samsung Galaxy) if the issue happens on mobile device**: Laptop
- **TensorFlow installed from (source or binary)**: Source
- **TensorFlow version (use command below)**: ('v1.9.0-rc2-2314-g3a99980', '1.10.0')
- **Python version**: 2.7
- **Bazel version (if compiling from source)**: 0.16.1
- **GCC/Compiler version (if compiling from source)**: 5.4.0 20160609
- **CUDA/cuDNN version**: no cuda
- **GPU model and memory**:  Geforce GT 730M, 1Gb memory
- **Exact command to reproduce**: 

`bazel-bin/tensorflow/compiler/aot/tfcompile --graph=/home/mehdi/Desktop/data/resnet101_model.pb --config=/home/mehdi/Desktop/data/config.pbtxt --cpp_class="foo::bar" --out_function_object="./out_model.o" --out_header="./out.h" --out_metadata_object="./out_helper.o"`

I have been testing [mask rcnn](https://github.com/matterport/Mask_RCNN) in order to do object detection and segmentation on images. It worked quite well and now I am trying to export my model to a standalone binary that can be used without having to install tensorflow on my target system. In order to do this I tried to use tfcompile. The first step was to take my .h5 keras model and converted it to a frozen tf .pb file. I used [this script](https://github.com/ericj974/Mask_RCNN/blob/master/scripts/export.py) that was especially written to convert mask rcnn models. I then downloaded the source code of tensorflow, compiled it by running

`bazel build --config=opt //tensorflow/tools/pip_package:build_pip_package --jobs 3`

before that, I set support XLA JIT to true when running configure.
After compilation was done I installed tensorflow and it was importable from python.

Then in order to compile my model I run:
`bazel-bin/tensorflow/compiler/aot/tfcompile --graph=/home/mehdi/Desktop/data/resnet101_model.pb --config=/home/mehdi/Desktop/data/config.pbtxt --cpp_class="foo::bar" --out_function_object="./out_model.o" --out_header="./out.h" --out_metadata_object="./out_helper.o"`

where resnet101_model.pb is the result of running export.py on my keras .h5 model.
config.pgtxt is as follows:

```
# Each feed is a positional input argument for the generated function.  The order
# of each entry matches the order of each input argument.  Here “x_hold” and “y_hold”
# refer to the names of placeholder nodes defined in the graph.
feed {
  id { node_name: "input_image" }
  shape {
    dim { size: 320 }
    dim { size: 320 }
    dim { size: 4 }
  }
}

# Each fetch is a positional output argument for the generated function.  The order
# of each entry matches the order of each output argument.  Here “x_y_prod”
# refers to the name of a matmul node defined in the graph.
fetch {
  id { node_name: "mrcnn_mask/Reshape_1" }
}
```

Unfortunately I can't upload my model as Github only allows 10Mb. But [here](https://github.com/matterport/Mask_RCNN/releases) is a similar model trained on the same network  (mask_rcnn_balloon.h5)

tfcompile fails with the following error:

`2018-08-16 12:38:27.078016: F tensorflow/compiler/aot/tfcompile_main.cc:150] Non-OK-status: status status: Not found: Op type not registered 'DenseToDenseSetOperation' in binary running on mehdi-t440p. Make sure the Op and Kernel are registered in the binary running in this process. Note that if you are loading a saved graph which used ops from tf.contrib, accessing (e.g.) tf.contrib.resampler should be done before importing the graph, as contrib ops are lazily registered when the module is first accessed. Aborted`

This is difficult to understand for me as DenseToDenseSetIOperation is existing and registered in tensorflow/core/ops/set_ops.cc (line 35).
I searched for tutorials or documentation, even watched the video presentation made by tensorflow guys, but there is no troubleshooting support anywhere. So I thought Github is the right place for this kind of problem.

**Comment by drpngx** [comment]
@tatatodd any hint?

**Comment by tensorflowbutler** [comment]
Nagging Assignee @drpngx: It has been 15 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Comment by tensorflowbutler** [comment]
Nagging Assignee @drpngx: It has been 45 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Comment by tensorflowbutler** [comment]
Nagging Assignee @drpngx: It has been 59 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Comment by tensorflowbutler** [comment]
Nagging Assignee @drpngx: It has been 75 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Comment by mehditlili** [comment]
Yes it is!

**Comment by tensorflowbutler** [comment]
Nagging Assignee @drpngx: It has been 15 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Comment by ileniTudor** [comment]
@drpngx. I still have the same problem

**Comment by haolujun** [comment]
I got the same problem too..

**Comment by sushreebarsa** [comment]
Hi There,

We are moving this issue to closed status, as you are using an older version of tensorflow(1.x) which is officially considered as end of life. We recommend that you upgrade to 2.4 or later version and let us know if the issue still persists in newer versions.
Please open a new issue for any help you need against 2.x, and we will get you the right help.Thanks!
